.github/workflows/build.yml

```
@@ -46,40 +69,179 @@ jobs:
         run: pre-commit install-hooks
       - name: Run pre-commit on all files
         run: pre-commit run --all-files
-  build:
+
+  prepare:
+    # Calculates and publishes outputs that are used by other jobs.
+    #
+    # Outputs:
+    #   created:
+    #     The current date-time in RFC3339 format.
+    #   repometa:
+    #     The json metadata describing this repository.
+    #   source_version:
+    #     The source version as reported by the `bump_version.sh show` command.
+    #   tags:
+    #     A comma separated list of Docker tags to be applied to the images on
+    #     DockerHub.  The tags will vary depending on:
+    #     - The event that triggered the build.
+    #     - The branch the build is based upon.
+    #     - The git tag the build is based upon.
+    #
+    #     When a build is based on a git tag of the form `v*.*.*` the image will
+    #     be tagged on DockerHub with multiple levels of version specificity.
+    #     For example, a git tag of `v1.2.3+a` will generate Docker tags of
+    #     `:1.2.3_a`, `:1.2.3`, `:1.2`, `:1`, and `:latest`.
+    #
+    #     Builds targeting the default branch will be tagged with `:edge`.
+    #
+    #     Builds from other branches will be tagged with the branch name. Solidi
+    #     `/` in branch names are replaced with hyphen-minuses `-` in the Docker
+    #     tag.
```

@jsf9k Glad you're enjoying the Solidus! 😝

Not sure if you saw the ```(`\`)``` on the next comment line...  Are you suggesting:
```suggestion
    #     Builds from other branches will be tagged with the branch name. Solidi
    #     ("slahes") in branch names are replaced with hyphen-minuses `-` in the Docker
    #     tag.  See: https://www.compart.com/en/unicode/U+002F
```
-or-
```suggestion
    #     Builds from other branches will be tagged with the branch name. Solidi
    #     ("slahes") `/` in branch names are replaced with hyphen-minuses `-` in the Docker
    #     tag.  See: https://www.compart.com/en/unicode/U+002F
```

---

I did miss the solidus that followed.  How about this:
```suggestion
    #     Builds from other branches will be tagged with the branch name. Solidi
    #     (`/` characters - commonly known as slashes) in branch names are replaced
    #     with hyphen-minuses (`-` characters) in the Docker tag.  For more information
    #     about the solidus see these links:
    #       * https://www.compart.com/en/unicode/U+002F
    #       * https://en.wikipedia.org/wiki/Slash_(punctuation)#Encoding
```

---

Love it.  Someone is going to wonder what makes this workflow tick, and they're gonna get a typography schooling for free!

---

There is a thang in thermodynamics called [_fugacity_](https://en.wikipedia.org/wiki/Fugacity).  It is also an English word that means "fleeting".  What I _most_ remember from learning about fugacity in grad school was the footnote at the bottom of the page where it was introduced in our textbook.  It was an excerpt from a quote by Samuel Johnson - "...the fugacity of pleasure, the fragility of beauty...".

Sometimes these asides really tie the room together.

---

> Add some additional comments in case anyone is confused about the nature of the solidus and its place within our tagging system.

What a great comment!

This whole exchange reminds me of [the Charles Portis book "Masters of Atlantis"](https://en.wikipedia.org/wiki/Masters_of_Atlantis) and Lamar Jimmerson and Sydney Hen's very serious study of the gnomon.  The solidus is the dev team's gnomon!